R/plotLoadings.R

```diff
@@ -0,0 +1,385 @@
+#' Plot feature loadings after performing a reducedDim 
+#'
+#' Inspired by the \code{\link[diffTop:plotASVcircular]{plotASVcircular}} method using phyloseq 
+#' and has been converted to use TreeSummarizedExperiment objects.
+#'
+#' @param x a
```

Comment:
TreeSummarizedExperiment or SingleCellExperiment? (the latter does not have a tree and it is more generic, TreeSE is a SCE).

Reply:
Yes, should have updated description but I added SCE support in last commits.